tests/core/test_datamodules.py

```diff
@@ -381,8 +381,8 @@ def _step(self, batch, batch_idx):
         def training_step(self, batch, batch_idx):
             _, y = batch
             out = self._step(batch, batch_idx)
-            out['loss'] = F.cross_entropy(out['logits'], y)
-            return out
+            loss = F.cross_entropy(out['logits'], y)
+            return loss
```

Comment:
just wondering why returning a dict doesn't work with dp. can you tell?

Reply:
discs shall be also supported... @SeanNaren?

Reply:
I think it should else it would be impossible to use metrics from PL while training in DP mode.